libtransmission/session.h

```diff
@@ -760,6 +760,7 @@ struct tr_session
         return open_files_;
     }
 
+    void flush_torrent_files(tr_torrent_id_t const tor_id) const noexcept;
```

Comment:
`const` has no effect here.

```suggestion
    void flush_torrent_files(tr_torrent_id_t tor_id) const noexcept;
```

P.S. I'm surprised this isn't caught by `clang-tidy`'s `readability-avoid-const-params-in-decls`... Might need to look into it.